README.md

```diff
@@ -89,36 +240,33 @@ Migrations
 
 To run database migrations:
 
-```
-$ make migrate
-```
+    make migrate
 
-The startup commands such as `make run` and `make rundev` check for pending migrations, and will 
-exit before starting the server if any are found. You can also check for pending migrations manually with:
+The startup commands such as `make run` and `make rundev` check for pending
+migrations, and will exit before starting the server if any are found. You can
+also check for pending migrations manually with:
+
+    make migration_check
 
-```
-$ make migration_check
-```
 
 Creating a user
 ---------------
 
-In order to login to the development server locally you will need to create a superuser through 
+In order to login to the development server locally you will need to create a
+superuser by running:
 
-```
-honcho run ./manage.py createsuperuser
-```
+    honcho run ./manage.py createsuperuser
+
+Once created, you will be able to login to with the username and password you
+set up.
 
```

Comment:
@omarkhan Seriously nitpicking:  "you will be able to login *to* with the username.." should be "you will be able to login with the username.."